features/bevragen/persoon/verificatie/dev/fields-gba.feature

```diff
@@ -29,34 +29,16 @@ Functionaliteit: verificatie velden vragen met fields (persoon)
       | overlijden.datum |
       | immigratie       |
 
-    Abstract Scenario: persoonsgegevens zijn geverifieerd en verificatie wordt wel gevraagd
+  Rule: verificatie mag niet worden gevraagd, omdat het automatisch wordt geleverd
+
+    @fout-case
+    Abstract Scenario: veld <fields> mag niet worden gevraagd, omdat het automatisch wordt geleverd
       Als gba personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 000000152                       |
+      | burgerservicenummer | 000000024                       |
```

Comment:
@MelvLee heb jij dit gewijzigd naar 000000024? Waarom? De Gegeven stap  voegt persoon met burgerservicenummer 000000152 toe

Reply:
laat maar, zat er al in (GitHub geeft wijzigingen bij een committee wat vreemd weer